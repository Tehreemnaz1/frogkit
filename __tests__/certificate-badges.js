import { mount } from 'vue-test-utils'
import Vue from 'vue'
import CertificateBadge from '@/components/certificate-badges'

describe('Certificate Badges component', () => {

  it('should render component and match snapshot', () => {
    const wrapper = mount(CertificateBadge)
    expect(wrapper.html()).toMatchSnapshot()
  })

})
