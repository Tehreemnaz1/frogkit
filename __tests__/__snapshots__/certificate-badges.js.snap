// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Certificate Badges component should render component and match snapshot 1`] = `<flex align="center" justify="space-around"><img width="65px" src="../assets/images/iso-27001.jpg"> <img width="65px" src="../assets/images/tuev.jpg"> <img width="65px" src="../assets/images/iso-9001.jpg"></flex>`;
